CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ Here we document changes that affect the public API or changes that needs to be
 ## 2019-03-06 
 New processor `Geometry Entry Exit Points` generates entry point and exit point images from any closed mesh to be used in raycasting. The positions of the input mesh are directly mapped to texture coordinates of a volume. This enables volume rendering within arbitrary bounding geometry.
 
+## 2019-02-24 Unified line rendering
```

Comment:
Should be moved up to the top of the changelog (i guess the git merge did it "wrong")